common.gypi

```diff
@@ -35,7 +35,7 @@
 
     # Reset this number to 0 on major V8 upgrades.
     # Increment by one for each non-official patch applied to deps/v8.
-    'v8_embedder_string': '-node.12',
+    'v8_embedder_string': '-node.13',
```

Comment:
This needs be bumped again as I've just landed https://github.com/nodejs/node/pull/32974 which bumped this to `.13`.
https://github.com/nodejs/node/blob/6419e59889e4498f4fe6515d58218a31a7d6578b/common.gypi#L38